src/main/java/org/killbill/billing/plugin/dao/payment/PluginPaymentDao.java

```diff
@@ -328,11 +331,23 @@ public Void withConnection(final Connection conn) throws SQLException {
                            .transaction(new TransactionalRunnable() {
                                @Override
                                public void run(final Configuration configuration) throws Exception {
+                                   PM_R record = DSL.using(conn, dialect, settings)
+                                      .selectFrom(paymentMethodsTable)
+                                      .where(DSL.field(paymentMethodsTable.getName() + "." + KB_PAYMENT_METHOD_ID).equal(kbPaymentMethodId.toString()))
+                                      .and(DSL.field(paymentMethodsTable.getName() + "." + IS_DELETED).equal(FALSE))
+                                      .and(DSL.field(paymentMethodsTable.getName() + "." + KB_TENANT_ID).equal(kbTenantId.toString()))
+                                      .orderBy(DSL.field(paymentMethodsTable.getName() + "." + recordIdFieldName).desc())
+                                      .fetchOne();
+
+                                   Field accountField = MoreObjects.firstNonNull(paymentMethodsTable.field(KB_ACCOUNT_ID), paymentMethodsTable.field(KB_ACCOUNT_ID.toLowerCase()));
```

Comment:
Why do you need this?

Reply:
record.get would fail if the column does not exist, and it is actually case sensitive.
The test would fail if it was upper KB_ACCOUNT_ID, but succeed in case of KB_ACCOUNT_ID.to_lower_case. To cover both cases where a plugin DDL use upper case for the account ID column name, I check for both cases.